.github/workflows/ci.yml

```diff
@@ -59,7 +71,7 @@ jobs:
         docs/_build/
         docs/generated/
         .tox/sample_data/
-      cache-key: docs-${{ github.run_id }}
+      cache-key: ${{ needs.compute-cache-key.outputs.cache-key }}
```

Comment:
There is another:
```
          cache-key: docs-${{ github.run_id }}
```
under the gallery build, is that fine to stay or does that need the same hash as this?